src/nimdowpkg/windowmanager.nim

```diff
@@ -542,6 +542,23 @@ proc jumpToUrgentWindow(this: WindowManager) =
   urgentMonitor.setSelectedTags(tagID)
   this.display.warpTo(urgentClient)
 
+template modWidthDiff(this: WindowManager, diff: int) =
+  var layout = this.selectedMonitor.taggedClients.findFirstSelectedTag.layout
+  if layout of MasterStackLayout:
+    var masterStackLayout = cast[MasterStackLayout](layout)
+    let screenWidth = masterStackLayout.calcScreenWidth(this.selectedMonitor.layoutOffset)
+    if (diff > 0 and masterStackLayout.widthDiff < 0) or
+      (diff < 0 and masterStackLayout.widthDiff > 0) or
+      masterStackLayout.calcClientWidth(screenWidth).int - abs(masterStackLayout.widthDiff).int - 10 > 0:
```

Comment:
I believe the 10 here is still hard-coded but shouldn't be?

Reply:
You're right